docs/user/README.md

```diff
@@ -430,6 +430,9 @@ eg:
 `box wrap blue air`
 Wraps the token containing letter 'a' with a blue hat in square brackets.
 
+The rewrap command can be used to swap a given pair of symbols with another
+- `"curly rewrap <TARGET>"`: wraps the target with curly brackets, replacing the previous paired delimiter
```

Comment:
I believe the [default](https://github.com/cursorless-dev/cursorless-vscode/blob/b9355460a8e602bf47ec0b5ff1e609ad2f9017b3/cursorless-talon/src/actions/actions.py#L89) is

```suggestion
- `"curly repack <TARGET>"`: wraps the target with curly brackets, replacing the previous paired delimiter
```

Reply:
I don't feel strongly about the heading. The internal identifier is rewrap so maybe let's keep that for the heading? Cc/ @AndreasArvidsson

Reply:
Agreed, the  identifiers is what I prefer to use since everybody has different spoken forms.